### PR TITLE
Move Bulk Ordering Tests to Live Tests

### PR DESCRIPTION
**Pull request by tvaron3**
# Description

The bulk ordering tests were running in the emulator, but they take a long time. Therefore, they were moved into the live tests instead of the emulator.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 #### Features Added
 * Added a flag for allowing customers to preserve ordering in bulk mode. See [PR 35892](https://github.com/Azure/azure-sdk-for-java/pull/35892)
-
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:7]: revert this please.

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java [hunk 2]
```diff
@@ -840,6 +859,189 @@ private void createItemsAndVerify(List<com.azure.cosmos.models.CosmosItemOperati
...
+        ImplementationBridgeHelpers.CosmosBulkExecutionOptionsHelper
+            .getCosmosBulkExecutionOptionsAccessor()
+            .setOrderingPreserved(cosmosBulkExecutionOptions, true);
+
+        FaultInjectionRule rule = injectBatchFailure("ServiceUnavailable", FaultInjectionServerErrorType.SERVICE_UNAVAILABLE, 1);
+
+        Flux<CosmosItemOperation> inputFlux = Flux
+            .fromIterable(cosmosItemOperations)
+            .delayElements(Duration.ofMillis(100));
+        final BulkExecutorWithOrderingPreserved<BulkExecutorTest> executor = new BulkExecutorWithOrderingPreserved<>(
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java:981]: Should this still be BulkExecutorTest? Shouldn't this be CosmosBulkAsyncTest?

**Review by kushagraThapar — CHANGES_REQUESTED** [review]

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java (hunk 2 above)
Comment (tvaron3) [reply sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java]: Changed to CosmosBulkAsyncTest

File: sdk/cosmos/azure-cosmos/CHANGELOG.md (hunk 1 above)
Comment (tvaron3) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md]: changed

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java [hunk 3]
```diff
@@ -4,30 +4,49 @@
 package com.azure.cosmos;
 
 import com.azure.cosmos.implementation.ImplementationBridgeHelpers;
+import com.azure.cosmos.implementation.batch.BulkExecutorTest;
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java:7]: nit: unused import.

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**